packages/drivers/odsp-driver/src/graph.ts

```diff
@@ -255,5 +255,5 @@ export async function getGraphItemLite(
         };
         graphItemLiteCache.add(cacheKey, valueGenerator);
     }
-    return graphItemLiteCache[cacheKey] as GraphItemLite;
+    return graphItemLiteCache.get(cacheKey);
```

Comment:
is it possible to test this?

Reply:
The TS compiler should have caught this, but we used an as escape hatch here. 

I'm very concerned that this appears to have no coverage (manual or automated in ff), @jatgarg - can you follow up here after we merge this PR?